org.kde.lokalize.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "org.kde.lokalize",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "lokalize",
+    "rename-icon": "lokalize",
+    "finish-args": 
+    [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kross",
+            "buildsystem": "cmake-ninja",
+            "config-opts": 
+            [ 
+                "-DENABLE_TESTING=OFF"
+            ],
+            "sources": 
+            [ 
+                { 
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/frameworks/5.69/portingAids/kross-5.69.0.tar.xz",
+                    "sha256": "acec70fe97dddd32b05b28747da7761571d8b684641c7848f544a0e67a698f93"
+                }
+            ]
+        },
+        {
+            "name": "hunspell",
```

Comment:
Do we need this?

If i do flatpak run org.kde.Sdk i can see /usr/lib/x86_64-linux-gnu/libhunspell-1.7.so is there so it seems hunspell is part of the SDK and we should not need to compile it?

Reply:
I'll check it

Reply:
https://github.com/flathub/flathub/pull/1603/commits/5057c7ae60aa3ad8892d5d486ee867be72622819